modules/openapi-generator/src/main/resources/rust/model.mustache

```diff
@@ -103,14 +95,6 @@ pub enum {{{enumName}}} {
 {{/enumVars}}
 {{/allowableValues}}
 }
-
-impl Default for {{{enumName}}} {
-    fn default() -> {{{enumName}}} {
-        {{#allowableValues}}
-        Self::{{ enumVars.0.name }}
-        {{/allowableValues}}
-    }
-}
```

Comment:
cc @Foorack who added these via https://github.com/OpenAPITools/openapi-generator/pull/10720 to fix regression introduced by https://github.com/OpenAPITools/openapi-generator/pull/10432 (authored by @PiDelport)

Reply:
yup, #15856 reverted #10432. this change reverts #10720.